ui/perfherder/alerts/AlertsView.jsx

```diff
@@ -121,6 +120,25 @@ class AlertsView extends React.Component {
     return pages;
   };
 
+  composeParams = (id, page, framework, status) => {
+    const params = id
+      ? { id }
+      : { framework: framework.id, page, status: summaryStatusMap[status] };
+
+    const doNotFilter = -1;
+    const listMode = !id;
+
+    // params used for UI purposes, not valid as API parameters
+    if (listMode && params.status === doNotFilter) {
+      delete params.status;
+    }
```

Comment:
I still find this comment confusing. Maybe it's the jet lag :) But, which are you referring to. I see `params.status` and `params.framework`. I think those are both used as query params correct?

Reply:
I see my old comment says "-1 ('all') is created for UI purposes but is not a valid API parameter".

Reply:
You're right. Restored the comment. It will refer now to `doNotFilter` variable.